src/pymatgen/symmetry/groups.py

```diff
@@ -166,6 +173,44 @@ def get_orbit(self, p: ArrayLike, tol: float = 1e-5) -> list[np.ndarray]:
                 orbit.append(pp)
         return orbit
 
+    def is_subgroup(self, supergroup: PointGroup) -> bool:
```

Comment:
Added the constraints due to crystallographic direction issue (I could think of) but would prefer a database solution similar to `SpaceGroup`. For example, it is an option to contact the Bilbao Crystallographic Server and ask for permission to use their database?

Reply:
If they release it with a license that allows us to use it, I guess it is an option.

Reply:
Not sure about that.